README.md

```diff
@@ -38,3 +72,36 @@ try {
 ```
 
 ### Contributing
+
+The takeaway for contributing is that feature branches are created off of the `develop` branch and pull requests should be made 
+into the `develop` branch rather than `master`. 
+
+For example, the workflow to create a pull request for a feature that adds support for fetch.txt follows
+
+- Create an issue describing your planned changes, or add a comment to an existing relevant issue
+- Checkout the `develop` branch, followed by `git checkout -b feature_support_fetch_file`
+- Commit your changes to the `feature_support_fetch_file` branch
+- Create a pull request from `feature_support_fetch_file` into `develop` and outline the code changes and how to test it
+- Once the code is reviewed, our team will merge in your changes and you're done!
+
+#### Code Style
+This project conforms to the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) and new 
+contributions should follow suite.
+
+#### Unit Tests
+Unit tests should be created for all new classes and if possible, public methods. When writing unit tests, keep in mind that 
+the points are 
+ - After your contribution is merged, no one accidentally breaks it in the future
+ - Early validation of potential bugs in your code
+ - Examples for future developers and users to see how the classes and methods can be used
+
+## Acknowledgments
+Work on this package was supported by:
+
+- NSF DIBBS grant [#1541450](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1541450) to B. Ludaescher, V. Stodden, N. Gaffney, M. Turk, and K. Chard
+
+Additional support was provided for working group collaboration by the National Center for Ecological Analysis and Synthesis, a Center funded by the University of California, Santa Barbara, and the State of California.
+
+[![nceas_footer](https://live-ncea-ucsb-edu-v01.pantheonsite.io/sites/default/files/2020-03/NCEAS-full%20logo-4C.png)](http://www.nceas.ucsb.edu)
```

Comment:
Link to the NCEAS header image should be https://www.nceas.ucsb.edu/sites/default/files/2020-03/NCEAS-full%20logo-4C.png